src/main/java/commands/ListCommand.java

```diff
@@ -0,0 +1,35 @@
+package commands;
+
+import tasks.Deadlines;
+import tasks.Events;
+import tasks.Task;
+import tasks.TaskList;
+import tasks.Todos;
+
+/**
+ * ListCommand prints out all the tasks in the task list.
+ */
+public class ListCommand extends Command {
+
+    /**
+     * Constructor for ListCommand.
+     */
+    public ListCommand() {
+      super();
+    }
+
+    @Override
+    /**
+     * Executes ListCommand by iterating through each task and printing out its string representation.
+     *
+     * @param taskList Task list to be printed out.
+     */
+    public void run(TaskList taskList) {
+        if (taskList.getSize() == 0) {
+            System.out.println("You have no tasks at the moment!");
```

Comment:
Not sure but maybe could put this under Ui? Since it is kind of a user interaction. Can then call Ui.noTasksMessage() or something like that.